src/containers/App.js

```diff
@@ -6,7 +6,7 @@ import logo from '../images/logo.jpg';
 import '../styles/App.css';
 import { APP_STATUS } from '../constants/status_types';
 import { startApp } from '../actions/core';
-
+import DevTools from '../components/common/DevTools';
```

Comment:
Wont the devtools get included in to production build's as well?

Reply:
Don't know yet, if users will want that. 
Otherwise we can save it as possible hint for future of application

Reply:
@jetpack3331 Can you maybe put it in an condition so that we have a way to exclude it? E.g. you can put an ENV variable and use that in the condition.

Reply:
@florianlenz NODE_ENV isn't good place to put in, better be some config.js file or parameter in CLI when building or running application

Reply:
@florianlenz Look on new commit